django/publicmapping/static/js/mapping.js

```diff
@@ -2425,7 +2425,14 @@ function mapinit(srs,maxExtent) {
                 row.append(swatch);
 
                 var title = $('<td/>');
-                title.append( rule.title );
+
+                var numberWithCommas = function(x) {
+                  return x.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+                }
+
+                newRuleTitle = rule.title.indexOf('.') === -1 ? rule.title :
+                    rule.title.substring(0, rule.title.indexOf('.'))
+                title.append( numberWithCommas(newRuleTitle) );
```

Comment:
Could probably remove the leading and trailing spaces inside the function call. I see that was how it was before you made the change too, but might as well tidy it up.